packages/clay-color-picker/package.json

```diff
@@ -0,0 +1,41 @@
+{
+	"name": "@clayui/color-picker",
+	"version": "3.0.0",
+	"description": "ClayColorPicker component",
+	"license": "BSD-3-Clause",
+	"repository": "https://github.com/liferay/clay/tree/master/packages/clay-color-picker",
+	"engines": {
+		"node": ">=0.12.0",
+		"npm": ">=3.0.0"
+	},
+	"main": "lib/ClayColorPicker.js",
+	"esnext:main": "src/ClayColorPicker.js",
+	"jsnext:main": "src/ClayColorPicker.js",
+	"files": [
+		"lib",
+		"src"
+	],
+	"scripts": {
+		"build": "babel src --root-mode upward --out-dir lib --extensions .ts,.tsx",
+		"build:types": "tsc --project ./tsconfig.declarations.json",
+		"prepublishOnly": "npm run build && npm run build:types",
+		"start": "webpack-dev-server --mode development",
+		"test": "jest --config ../../jest.config.js"
+	},
+	"keywords": [
+		"clay",
+		"react"
+	],
+	"dependencies": {
+		"@types/tinycolor2": "^1.4.1",
```

Comment:
I'm not sure if we should add as a dependency. I do not know what the pattern is out there about the types... Can you check this?

Reply:
Should probably be a `devDependency`; these packages don't do anything at runtime.